cfgov/unprocessed/js/routes/on-demand/feedback-form.js

```diff
@@ -4,39 +4,60 @@
 
 'use strict';
 
+var messages = {
```

Comment:
This should go in the config file. The on-demand routes were always meant to just be used for minimal code initialization. 

If you want to add more advanced logic I would suggest moving the code and making it reusable.